### ISSUE TITLE
players "delay"

### ISSUE THREAD

**Issue by Traveler-Valhalla** [Valhalla-Gaming/Tracker#8271]
**Describe the issue you're having**:
"Some" players in battlegrounds are "jumping/teleporting" around instead of walking normally. Because of this you can't melee hit them even when they are infornt of you or you can't cast a spell because - apparently - they are either behind you or too far away.
There's also some "delay" with their "position". When you - for example - stun them they will still move for a while.
Also when you fear players in battleground they will "run in terror" to one direction and then "teleport" to another when its effect ends. So for example you can fear them, start casting a spell and get error message near the end that "Target needs to be infront of you." because they "teleport" somewhere behind you.

I'm having 40-50ms so I guess it's not my fault.

**Comment by MrNonameNo3** [comment]
I have the feeling every player has that...

**Comment by Milofajjour** [comment]
It's a common bug for all

**Comment by yebadal** [comment]
yea fuckiing lag

**Comment by GM-Pyromanic** [comment]
Please do not use language like that on the issue tracker.

**Comment by spenzostolen** [comment]
OK.
There has been these issue since I've started to play the server(not sure it wasn't even before), and from my perspective as a melee player, this is very disturbing.
I constantly see this phenomenon on battlegrounds and arenas.
When an [b]enemy[/b] uses some sort of a teleport, a displacer, a speed buff for a short duration it just lags him out and does not properly update his position to the server, to ther other players making it very hard to follow the person, considering he has some sort of a buff or a skill.
Another issue is when you are on a target, it sometimes execuetes the animation, but not the skill itself.
Maybe it's an animation miscalculation of yards or something, but I suggest checking that out too.
